modules/virt-configuring-cluster-dpdk.adoc

```diff
@@ -150,3 +152,19 @@ spec:
   nodeSelector:
     feature.node.kubernetes.io/network-sriov.capable: "true"
 ----
+
+. Optional: If you created a custom machine config pool for your high-availability cluster in step 2:
```

Comment:
I think its not clear enough that these are removal instructions since they follow the setup instructions.
Please consider moving it under a title / heading.

Reply:
I have pulled those instructions out into a separate procedure: https://84850--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-using-dpdk-with-sriov.html#virt-removing-custom-mcp_virt-using-dpdk-with-sriov